### ISSUE TITLE
Should we create an Emeritus status for prior AB/TAG/BoD members

### ISSUE THREAD

**Issue by tobie** [w3c/w3process#689]
In #688, @chaals mentioned the following:

> However, I think there are better mechanisms available than ensuring some people get to hang around when the membership have hanged their mind about what they are looking for.

> As an ex-member of the AB, and a member of W3C, I'm perfectly able to provide my input - whether asked for it or not. And like others in that position, I do so. Groups can, and sometimes does, ask specific people outside the group for input, if they feel it would be helpful.

This echoes a different conversation about giving AB and TAG members access to the w3c-ac-forum@ mailing list after their term is over even if they aren't an AC Rep.

So here's my suggestion: what if we created some kind of emeritus status for past AB, TAG, and BoD members, with full access to w3c-ac-forum@ and member-only GitHub repos, perhaps with an additional dedicated mailing list, and with a sense that they're providing in exchange some form of custody of institutional memory and can act as a resource for current governing bodies in exchange?

**Comment by chaals** [comment]
There is currently some ad-hoc process by which individuals are granted a status like this. I think it makes a lot of sense to provide ongoing member access to all former members of the TAG, AB or BoD given an individual agreement to respect confidentiality (including not sharing member-confidential information with a non-member organisation that happens to be your employer), patent policy, CEPC, and so on.

**Comment by michaelchampion** [comment]
I might be the only current person whose access to W3C resources comes from "emeritus" status on the AB.  Other AB veterans such as Chaals and Tantek are AC reps so don't need this special status.

I don't think it's so much ad-hoc as the AB having resolved to advise W3M to grant people in my situation access to AC Forum. But I had to affiliate (as a volunteer) with a W3C member organization to get access to member-only resources.  If "emeritus" status is formalized, it should explicitly include at least read-only access to member-only resources.

I also like @tobie 's suggestion to treat the "emeritus" group as a resource for institutional memory, mentoring prospective AB/TAG members, being available to advise the Board /AB/TAG on issues we were once involved in, etc.

**Comment by tobie** [comment]
There's a lot of AB and TAG alumni:

- https://www.w3.org/2002/ab/
- https://tag.w3.org/history/

**Comment by mnot** [comment]
This is find and good, but I'd much rather establish a _real_ community discussion forum that's open, so that it isn't necessary.

**Comment by michaelchampion** [comment]
Agree with @mnot . I’m more interested in a forum for FUTURE TAG and AB members to build the necessary skills while sharing fresh perspectives

**Comment by tobie** [comment]
> This is find and good, but I'd much rather establish a real community discussion forum that's open, so that it isn't necessary.

Are you suggesting making most of what's currently member-privileged information public? I'd be very much in favor of that, obviously.

**Comment by dwsinger** [comment]
I don't think it is either/or. We can get better value from past AB and TAG members, and also get better community engagement. Can we split the latter as a separate issue?

**Comment by tobie** [comment]
>  Can we split [community engagement] as a separate issue?

Increased community (AC?) engagement seems like a much broader issue than just a process one (although there are process components—e.g. a year round chair, better tooling, etc.). Where would you want it opened?

**Comment by chaals** [comment]
There are 38 AB Alumni, of whom at least 6 currently have member access. There are 36 TAG alumni, of whom again at least 6 currently have member access. At least one person who is an alumnus of both, but as far as I know does not have member access.

So we're talking about granting access to about 60 people from the last 20-odd years. A large proportion of them are retired, and may or may not choose to take up that access (as are some of those who actually have access).

With slightly larger AB and TAGs than in the early days, and a BoD, I would expect an increase in new alumni over the next two decades. But 200 more people with member access seems like a drop in the ocean compared to the number eligible for it (400-odd members, a number with tens of thousands of employees). Helping AC reps draw on the memory of those who choose to provide it seems a good deal in exchange for providing this access.
